packs/equipment/headbands-of-translocation.json

```diff
@@ -34,7 +34,36 @@
             "title": "Pathfinder Treasure Vault"
         },
         "quantity": 1,
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.Equipment.HeadbandsOfTranslocation.ToggleLabel",
+                "option": "headbands-of-translocation",
+                "toggleable": true
+            },
+            {
+                "adjustment": {
+                    "criticalFailure": "to-failure"
+                },
+                "key": "AdjustDegreeOfSuccess",
+                "predicate": [
+                    "action:aid",
+                    "headbands-of-translocation"
+                ],
+                "selector": "skill-check"
+            },
+            {
+                "adjustment": {
+                    "criticalFailure": "to-failure"
+                },
+                "key": "AdjustDegreeOfSuccess",
+                "predicate": [
+                    "action:aid",
+                    "headbands-of-translocation"
+                ],
+                "selector": "attack-roll"
```

Comment:
I don't think it is possible in the system to have an aid roll on an attack, since we can't do inlines. Maybe drop the aid on this since it's implicit in the other roll option?